src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsMedrecReloading.java

```diff
@@ -25,6 +25,9 @@ public void regularFloatingMedrecIsReloadingWithRecirculationModule() {
     Page page = new Page("project43", "SyntheticTests/LongPage");
     AdsBaseObject ads = new AdsBaseObject(driver, page.getUrl());
 
+    driver.manage().getCookies().stream().forEach(c -> System.out.println(c.getName() + " = " + c.getValue() +
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this usage of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS106)